apps/common/src/python/mediawords/util/config/common.py

```diff
@@ -155,6 +155,15 @@ def password() -> str:
         """Password."""
         return ''
 
+    @staticmethod
+    def unsubscribe_address() -> str:
+        """Mailto link for email to which unsubscribe/account deletion requests should be sent"""
+        address = env_value('MC_EMAIL_UNSUBSCRIBE', required=False, allow_empty_string=True)
+        if address is None or '@' not in address:
+            address = 'support@example.com'
+        return address
+        # return f'mailto:{address}?subject=Delete%20account%20and%20unsubscribe'
```

Comment:
Could you remove this commented out line?

Reply:
🙈